integration-tests/chopsticks/src/tests/index.ts

```diff
@@ -32,13 +34,22 @@ beforeAll(async () => {
 		polkadotContext.dev.newBlock(newBlockConfig),
 		spiritnetContext.dev.newBlock(newBlockConfig),
 		hydradxContext.dev.newBlock(newBlockConfig),
+		assetHubContext.dev.newBlock(newBlockConfig),
 	])
 }, 300_000)
 
 afterAll(async () => {
-	// fixes api runtime disconnect warning
-	await setTimeout(50)
-	await Promise.all([spiritnetContext.teardown(), hydradxContext.teardown(), polkadotContext.teardown()])
+	try {
```

Comment:
What's the benefit of this `try-catch` block?

Reply:
Chopsticks is sometimes throwing errors by trying to shut down a chain. In this stage, errors doesn't matter anymore. That's the reason behind the try catch